.idea/runConfigurations/_Generate_FIR_Checker_Boilerplate.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name=" Generate FIR Checker Boilerplate" type="GradleRunConfiguration" factoryName="Gradle">
```

Comment:
Is this intentionally to start the name with a space? `name=" Generate..."`

Reply:
Oops! Nope, not at all.